schemas/kits-schema.json

```diff
@@ -35,6 +35,22 @@
                 "type": "string",
                 "description": "Architecture to target"
             },
+            "environmentVariablesShellScript": {
+                "type": "string",
+                "description": "Filename of a shell script which sets environment variables for the kit"
```

Comment:
I would recommend making this always be treated as an absolute path instead of just a file name.  Can you update the description to clarify this?

I would also like to name this something else.  Perhaps `environmentSetupScript` or `environmentConfigurationScript`?  Then the description can say "The absolute path to a script that modifies the environment for the Kit"

Reply:
Yes, environmentSetupScript and "The absolute path to a script that modifies the environment for the Kit" sounds better.